build.yml

```diff
@@ -0,0 +1,68 @@
+parameters:
+  name: ''
+  queue: ''
+  platform: ''
+
+phases:
+
+- phase: ${{ parameters.name }}
+  queue:
+    name: ${{ parameters.queue }}
+    parallel: 3
+    matrix:
+      Python35:
+        PYTHON_VERSION: '3.5'
+      Python36:
+        PYTHON_VERSION: '3.6'
+      ${{ if eq(parameters.platform, 'Linux') }}:
+        Python37:
+          PYTHON_VERSION: '3.7'
+      # Python38-dev:
+      #   PYTHON_VERSION: '>= 3.8.0a'
+
+  steps:
+
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(PYTHON_VERSION)'
+        architecture: 'x64'
+      displayName: Use Python version $(PYTHON_VERSION)
+    
+    - ${{ if eq(parameters.platform, 'Linux') }}:
+      - bash: |
+          sudo apt-get -yq update
+          sudo apt-get -yq install libmemcached-dev
+        displayName: Setup Linux environment
+
+    - script: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install unittest-xml-reporting
+      displayName: Setup Python environment
+
+    - script: |
+        pushd tests

```

Comment:
Out of interest, why is the `pushd` needed here? Couldn't you just `pip install -r tests/requirements/..` and run `python tests/runtests.py ...`?

Reply:
I think I copied it from the instructions for developers to run tests locally (not a literal copy-paste - I think the instructions were to `cd tests`). I can re-evaluate whether it's needed.